src/OrchardCore/OrchardCore.Search.Elasticsearch.Core/Recipes/ElasticsearchIndexStep.cs

```diff
@@ -26,20 +26,31 @@ ElasticsearchIndexManager elasticIndexManager
 
     protected override async Task HandleAsync(RecipeExecutionContext context)
     {
-        if (context.Step["Indices"] is not JsonArray jsonArray)
-        {
-            return;
-        }
+        // Elasticserach uses the term "Indices" for the plural of index, but the common spelling in US English is
+        // "Indexes". So both are supported to avoid unnecessary spelling problems.
+        var indexes = context.Step["Indexes"] as JsonArray ??
+                context.Step["Indices"] as JsonArray ??
+                [];
```

Comment:
previous syntax is much more readable. Please revent

Reply:
How would you check for both "Indexes" and "Indices" using the previous syntax? Something like this, I think.
```csharp
        if (context.Step["Indices"] is not JsonArray jsonArray)
        {
                if (context.Step["Indexes"] is not JsonArray indexesJsonArray)
                {
                    return;
                }

                jsonArray = indexesJsonArray;
        }
```

But in my opinion this is much less readable.

Reply:
Why do we need both? the docs show to use `Indices` not `indexes`. Why do we need to change it?

Reply:
As explained above, the correct American spelling ([also the one preferred by Microsoft](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/index-indexes-indices)) is Indexes, so this is an unnecessary source of spelling confusion.

Reply:
Thanks, I will go in that direction then.